src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/brokers.go

```diff
@@ -105,6 +105,13 @@ leader handles the request.
 			cl, err := admin.NewClient(fs, cfg)
 			out.MaybeDie(err, "unable to initialize admin client: %v", err)
 
+			b, err := cl.Broker(cmd.Context(), broker)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			if b.Maintenance.Draining {
+				out.Die("node cannot be decommissioned while it is in maintenance mode. Take the node out of maintenance mode first by running 'rpk cluster maintenance disable %v'", broker)
```

Comment:
Since this is a multi-sentence error, and it's the output we want before exit-1'ing, can / should we format this slightly better?
```
out.Die(`Node cannot be decommissioned while it is in maintenance mode.
Take the node out of maintenance mode first by running:

    'rpk cluster maintenance disable %v'
`, broker)
```